### PR TITLE
Feature/new knolx sessions frontend automation testing

### PR DESCRIPTION
**Pull request by JuliKumari-NashTech**

### CODE REVIEW

**Review by Ritika-gupta20 — APPROVED** [review]

File: go1percent/test/Go1PercentFEAutomation/KNOLXSESSIONS/knolxSessions.js
Comment (Ritika-gupta20) [inline go1percent/test/Go1PercentFEAutomation/KNOLXSESSIONS/knolxSessions.js:1]: Name of folder to be in camel case

File: go1percent/test/Go1PercentAPIAutomation/KNOLXSESSIONS/knolxSessions.js
Comment (Ritika-gupta20) [inline go1percent/test/Go1PercentAPIAutomation/KNOLXSESSIONS/knolxSessions.js:1]: Name of folder to be in camel case

**Review by Ritika-gupta20 — CHANGES_REQUESTED** [review]
Camel casing to be used to name folders

File: go1percent/test/Go1PercentFEAutomation/KNOLXSESSIONS/knolxSessions.js
Comment (JuliKumari-NashTech) [reply go1percent/test/Go1PercentFEAutomation/KNOLXSESSIONS/knolxSessions.js:1]: changes done

**Review by Ritika-gupta20 — APPROVED** [review]

**Comment by Ritika-gupta20** [comment]
conflicts in conf.js

**Review by Sparsh79 — APPROVED** [review]

**Comment by JuliKumari-NashTech** [comment]
> conflicts in conf.js

**Comment by JuliKumari-NashTech** [comment]
resolved conflict

**Pull request merged**